websites/A/Aniorb/presence.ts

```diff
@@ -0,0 +1,182 @@
+const presence = new Presence({
+    clientId: "776312991939428373"
+  }),
+  strings = presence.getStrings({
+    play: "presence.playback.playing",
+    pause: "presence.playback.paused"
+  }),
+  browsingStamp = Math.floor(Date.now() / 1000);
+
+let timestamps: number[],
+  video: HTMLVideoElement,
+  currentTime: number,
+  duration: number,
+  paused: boolean,
+  iFrameVideo: boolean,
+  playback: boolean;
+presence.on(
+  "iFrameData",
+  (data: {
+    iframe_video: {
+      duration: number;
+      iFrameVideo: boolean;
+      currTime: number;
+      paused: boolean;
+    };
+  }) => {
+    playback = data.iframe_video.duration !== null ? true : false;
+    if (playback) {
+      iFrameVideo = data.iframe_video.iFrameVideo;
+      currentTime = data.iframe_video.currTime;
+      duration = data.iframe_video.duration;
+      paused = data.iframe_video.paused;
+    }
+  }
+);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "aniorb",
+    startTimestamp: browsingStamp
+  };
+  if (document.location.pathname == "/") {
+    presenceData.details = "Browsing Aniorb";
+  } else if (document.location.pathname.includes("/recentlyadded")) {
+    presenceData.details = "Browsing Recently Added";
+    presenceData.smallImageKey = "recent";
+    presenceData.buttons = [
+      {
+        label: "View Recently Added",
```

Comment:
This label doesn't make any sense.

Reply:
Wdym it doesn't make any sense? Can you elaborate please?

Reply:
View recently added what... Site? Subpage?

Reply:
Takes you to the website's Recently Added page?

Reply:
sub page

Reply:
Okay but this label still doesn't make any sense with only these 3 words.